lib/reverse_sentence.rb

```diff
@@ -1,4 +1,66 @@
 # A method to reverse the words in a sentence, in place.
 def reverse_sentence(my_sentence)
-  raise NotImplementedError
+  if my_sentence == nil || my_sentence == ""
+    return nil
+  else
+    reverse_words(my_sentence)
+    string_reverse(my_sentence)
+  end
+  return my_sentence
```

Comment:
Similar to the previous two string manipulation assignments, reverse_sentence is manipulating the input string in place and hence you don't need to return anything. Lines 4 and 9 could simply be `return`